sdk/test-utils/testing-recorder-new/package.json

```diff
@@ -17,14 +17,14 @@
     "clean": "rimraf dist dist-esm test-dist types *.tgz",
     "extract-api": "echo skipped",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "mocha -r esm --require ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace \"test/*.spec.ts\"",
+    "integration-test:browser": "dev-tool run test:browser",
+    "integration-test:node": "dev-tool run test:node-ts-input --mocha=\"--timeout 1200000 'test/*.spec.ts'\"",
```

Comment:
Would it make sense to have a separate command without timeout? I have found it useful in the past, I wonder if it would be useful here, or in general (outside of this line I’m pinning with this comment), to possibly run the new recorder “without timeouts”.

Reply:
Unrelated to this PR. It would be a broader discussion irrespective of the dev-tool extension that this PR does.

If you want the timeout, call as `"dev-tool run test:node-ts-input --mocha=\"--timeout 1200000 'test/*.spec.ts'\""`.
If you don't want the timeout, call `"dev-tool run test:node-ts-input --mocha=\"'test/*.spec.ts'\""` or just `"dev-tool run test:node-ts-input`.

The same applies to any of the options that you'd pass to the mocha command.

Reply:
of course, of course! Thanks for following up! Approving